src/main/java/homework_3/MyImmutableClass.java

```diff
@@ -0,0 +1,55 @@
+package homework_3;
+
+import java.time.LocalDate;
+
```

Comment:
Please write here the comments with requirements for immutable class

Reply:
what you mean?